### ISSUE TITLE
Nightmare persist partition on windows bug

### ISSUE THREAD

**Issue by daniel3d** [segmentio/nightmare#755]
I was having problems to get Nightmare and Electron app to share a [persist partition](https://github.com/segmentio/nightmare/issues/754)

It turn out is a bug for some reason in windows environment you need to use app.quit() in order to save the persist partition by adding this code

```

/**
 * Make sure we close the app
 */
app.on('window-all-closed', function () {
  // On OS X it is common for applications and their menu bar
  // to stay active until the user quits explicitly with Cmd + Q
  if (process.platform !== 'darwin') {
    app.quit()
  }
})

/**
 * Listen for the app being "ready"
 */
app.on('ready', function() {
  var win, frameManager, options, closed; 

......

```

to runner.js it fixes the problem it may be related with other issues

**Comment by daniel3d** [comment]
Actually only this don't fix the problem it seams the way Nightmare is kilning the process on .end() is not the best way of closing electron process.

ignore the first comment code i posted this one seams to work i cannot find any issue with Nightmare

in runner.js after line 69

```
  /**
   * Exit the electron app correctly
   */
  parent.respondTo('quit', function(done) {
    app.quit(); done();
  });
```

and in the nightmare.js replace line 211 with this
```
    // instead instance.proc.kill(); use quit
    instance.child.call('quit', () =>{
      instance.child.removeAllListeners();
    });
```

**Comment by rosshinkley** [comment]
This should have been fixed with #757, I think.  Closing.
